packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -592,13 +594,22 @@ export const BREAKOUTS = {
   DEFAULT_TTL: 30,
 };
 
+export const INTERPRETATION = {
+  EVENTS: {
+    SUPPORT_LANGUAGES_UPDATE: 'SUPPORT_LANGUAGES_UPDATE',
+  },
+  INTERPRETER_CONTROL_SI_SUPPORTED: 'INTERPRETER_CONTROL_SI_SUPPORTED',
+  HOST_CONTROL_SI_SUPPORTED: 'HOST_CONTROL_SI_SUPPORTED',
```

Comment:
These are no longer needed.  please remove.
The `deviceCapabilities` will be used directly in the client.

Reply:
@sundayu OK